topics/statistics/tutorials/iwtomics/tools.yaml

```diff
@@ -0,0 +1,10 @@
+---
+api_key: admin
+galaxy_instance: http://localhost:8080
+install_resolver_dependencies: true
+install_tool_dependencies: true
+
+tools:
+  - name: suite_iwtomics
```

Comment:
Is it possible to add each tool individually? We can not check which Galaxy instance has the tools with a suite 😟

Reply:
If you have the workflow exported, you can generate that easily (more details [here](http://galaxyproject.github.io/training-material/topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html#filling-the-toolsyaml))

Reply:
Something like this?
```
tools:
  - name: iwtomics_loadandplot
    owner: iuc
    tool_panel_section_label: "IWTomics"
    
  - name: iwtomics_testandplot
    owner: iuc
    tool_panel_section_label: "IWTomics"

  - name: iwtomics_plotwithscale
    owner: iuc
    tool_panel_section_label: "IWTomics"
```

Reply:
yes :)